addons/web/static/src/views/fields/properties/kanban_properties_field.scss

```diff
@@ -1,3 +1,9 @@
 .o_kanban_property_field {
     max-width: 300px;
+    font-size: 0.875rem;
+
+    .form-check{
```

Comment:
(Linting: Also missing a space after the rule name and before the opening bracket)